std_msgs/Header header
std_msgs/ColorRGBA color
geometry_msgs/Point centroid
sensor_msgs/RegionOfInterest bounding_rectangle


